### DISCUSSION TITLE
Cannot start asusrouter

### DISCUSSION THREAD

**Discussion by deefadamo** [Vaskivskyi/ha-asusrouter#584] [Q&A]
Hi,
I am getting this error when adding asusrouter via `custom_components/` folder:

```
Validate your configuration if you recently made some changes to it and want to make sure that it is all valid.
Configuration invalid!

Component error: asusrouter - cannot import name 'UnitOfDataRate' from 'homeassistant.const' (/usr/src/homeassistant/homeassistant/const.py)
```

I have tried the lastet 2 releases, and the `stable` branch, all give the same error.
I'm running HomeAssistant in a docker container (on a NAS, not that this should affect anything)

The YAML block in `configuration.yaml` is as simple as it gets:
```
asusrouter:
  - platform: asusrouter
```

It obviously finds and starts to load the integration before hitting the `import` error.

Hopefully this is a PEBCAK issue and someone can point me in the right direction.   Thanks for any assistance.

**Comment by Vaskivskyi** [comment]
Hello, @deefadamo,

You don't need to put anything into the YAML configs. The integration is configurable through the Configuration flow in the HA UI

![image](https://user-images.githubusercontent.com/6554415/229336011-5241d965-7dfb-47e0-b418-ef0f032db26c.png)

If the issue still persists when configuring from the UI, please let me know the HA version you are using, so that I can check it out

**Reply by deefadamo** [reply]
Hi @Vaskivskyi 

Sorry, I did forget to mention that critical item - it is not able to be found when searching as per the Configuration flow in the UI.
After not finding it, I was trying to set it in `configuration.yaml` to see if it was found in `custom_components/` directory - which it was, but gave the error I saw.

So perhaps my issue is not with asusrouter, and I should ask elsewhere.
FYI I am running `:latest` HA from docker hub.
